packages/web-runtime/tests/unit/components/Topbar/__snapshots__/UserMenu.spec.js.snap

```diff
@@ -29,8 +29,10 @@ exports[`User Menu component when basic quota is set renders a danger quota prog
       <li class="storage-wrapper oc-pl-s">
         <oc-icon-stub name="cloud" filltype="line" accessiblelabel="" type="span" size="medium" variation="passive" color="" class="oc-p-xs"></oc-icon-stub>
         <div class="storage-wrapper-text">
-          <p class="oc-my-rm"><span>Personal storage (91.0% used)</span> <br> <span class="oc-text-small">910 B of 1 kB used</span></p>
-          <oc-progress-stub value="91" max="100" size="small" variation="danger"></oc-progress-stub>
+          <p class="oc-my-rm">
+            <!----> <span class="oc-text-small">910 B of 1 kB used</span>
```

Comment:
Is this a comment? A line comment or until the next occurrence? Looks fishy

Reply:
It's the `<template v-if="xy" .. />` which doesn't fulfill the condition and is hence rendered as an "empty comment" in snapshot tests